src/pages/dashboard/Dashboard.tsx

```diff
@@ -1,55 +1,173 @@
+  justify-items: center;
+  gap: 1rem;
+`;
+
+const StyledText = styled.div`
+  grid-area: text;
+  text-align: center;
+`;
+
+const StyledTagline = styled(Typography)`
+  font-family: 'Barlow', sans-serif;
+  font-weight: bold;
+  max-width: 40rem;
+  grid-area: text-tagline;
+  white-space: pre-wrap;
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    font-size: 2rem;
+  }
+`;
+
+const StyledShortDescription = styled(Typography)`
+  padding-top: 1.5rem;
+  max-width: 40rem;
+  grid-area: short-description;
+  white-space: pre-wrap;
+`;
+
+const StyledCollapse = styled(Collapse)`
+  grid-area: text-description;
+  padding-top: 1rem;
+`;
+
+const StyledButtonWrapper = styled.div`
+  grid-area: button;
 `;
 
 const Dashboard = () => {
-  const { t } = useTranslation();
-  const history = useHistory();
+  const { t } = useTranslation('common');
+  const [readMore, setReadMore] = useState(false);
 
-  const handleSearch = async (searchTerm: string) => {
-    if (searchTerm.length) {
-      history.push(getSearchPath(searchTerm));
-    }
-  };
+  const toggleReadMore = () => setReadMore(!readMore);
 
   return (
     <StyledDashboard>
-      <StyledAbout>
-        <Card>
-          <Typography variant="subtitle1">{t('about:short_description')}</Typography>
-          <MuiLink component={Link} to={UrlPathTemplate.About} data-testid="about_read_more_link">
-            {t('common:read_more')}
-          </MuiLink>
-        </Card>
-      </StyledAbout>
-      <StyledSearchBarContainer>
-        <SearchBar handleSearch={handleSearch} initialSearchTerm="" />
-        <LatestRegistrations />
-      </StyledSearchBarContainer>
+      <StyledTaglineDiv backgroundColor={lightTheme.palette.section.megaDark}>
+        <StyledTagline variant="h1">{t('nva_tagline')}</StyledTagline>
+        <StyledShortDescription variant="h3" variantMapping={{ h3: 'p' }}>
+          {t('about:short_description')}
+        </StyledShortDescription>
+      </StyledTaglineDiv>
+      <StyledDescriptionDiv backgroundColor={lightTheme.palette.section.megaDark}>
+        <StyledCollapse in={readMore} collapsedHeight="0rem">
+          <AboutContent />
+        </StyledCollapse>
+        <StyledButtonWrapper>
+          <Button color="secondary" variant="contained" data-testid="button-read-more" onClick={toggleReadMore}>
+            {t(readMore ? 'read_less' : 'read_more')}
+          </Button>
+        </StyledButtonWrapper>
+      </StyledDescriptionDiv>
+      <StyledLinksContainer>
+        <StyledSearchLink to={UrlPathTemplate.Search}>
+          <StyledLinkContent>
+            <SearchIcon fontSize="large" />
+            <StyledText>
+              <Typography variant="h4" variantMapping={{ h4: 'p' }}>
+                {t('search_for_publication')}
+              </Typography>
+            </StyledText>
+          </StyledLinkContent>
+        </StyledSearchLink>
+        <StyledNewRegistrationLink to={UrlPathTemplate.NewRegistration}>
```

Comment:
```suggestion
        <StyledNewRegistrationLink to={getRegistrationPath()}>
```
er det samme, men potet potitt osv

Reply:
ahh, deet var ikke logisk for meg da jeg gjorde dette.